packages/lib/Logger.ts

```diff
@@ -287,6 +291,45 @@ class Logger {
 		}
 	}
 
+	private cleanLogFile(logFileFullPath) {
```

Comment:
This is causing a compilation error! See [the CI log](https://github.com/laurent22/joplin/actions/runs/5392723686/jobs/9791481595?pr=8376#step:12:161).

```suggestion
	private cleanLogFile(logFileFullPath: string) {
```

To fix this, be sure to give parameters type annotations.

This might also need to be done for the other `private` functions with arguments.